tailwind.config.js

```diff
@@ -1,13 +1,32 @@
+import { addIconSelectors } from '@iconify/tailwind'
+
 /** @type {import('tailwindcss').Config} */
 export default {
-	content: ['./src/**/*.{html,js,svelte,ts}'],
-	theme: {
-		extend: {
-			transitionProperty: {
-				height: 'height, max-height',
-				margin: 'margin'
-			}
-		}
-	},
-	plugins: []
-};
+  content: ['./src/**/*.{html,js,svelte,ts}'],
+  theme: {
+    extend: {
+      transitionProperty: {
+        height: 'height, max-height',
+        margin: 'margin',
+      },
+    },
+  },
+  plugins: [
+    addIconSelectors({
```

Comment:
this setup is for iconify when using tailwind.
and i made the icons bolder to match how they were displayed previously